tests/cargo-kani/stubbing-validate-random/src/main.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright Kani Contributors
```

Comment:
I don't this test is being run because it doesn't have an `main.expected` file.

Reply:
BTW, you can just add a `expected` file where `Cargo.toml` lives and compiletest will run `cargo kani` without `--harness` and compare the output.

Reply:
Thanks! I took care of this